supabase/functions/billing/index.ts

```diff
@@ -19,18 +20,15 @@ serve(async (req) => {
             const request = await req.json();
 
             const requested_tenant = request.tenant;
+
+            const authHeader = req.headers.get("Authorization");
+            if (!authHeader) {
+                throw new Error("Missing headers");
+            }
```

Comment:
This probably should never happen... but since I was in here I decided to make things a tiny bit safer.